src/dropdown.test.interactions.filterable.ts

```diff
@@ -504,30 +486,6 @@ it('does not filter on only whitespace', async () => {
   expect(options.length).to.equal(2);
 });
 
-it('hides the options when all of them are filtered out', async () => {
```

Comment:
Cruft. This test has been obviated by [these](https://github.com/CrowdStrike/glide-core/pull/929/files#diff-749aecf3d8907f729dad99b541e170b750aeed9ebcfabe5a57b850521515213cR1905) [assertions](https://github.com/CrowdStrike/glide-core/pull/929/files#diff-749aecf3d8907f729dad99b541e170b750aeed9ebcfabe5a57b850521515213cR1938).